docker/README.md

```diff
@@ -182,31 +186,13 @@ spec:
 
 Like the other solutions, remember to replace device `/dev/ttyACM0` with the path of your USB stick and choose the solution you prefer for data persistence.
 
-### ATTENTION
-
-If you get the error `standard_init_linux.go:207: exec user process caused "exec format error"` probably it's because you previously installed a wrong architecture version of the package so in that case you must delete the existing volume that contains the old executable:
-
-`docker volume rm zwavejs2mqtt`
-
-Check files inside volume
-
-```bash
-docker run --rm -it --mount source=zwavejs2mqtt,target=/usr/src/app zwavejs/zwavejs2mqtt:latest find /usr/src/app
-```
-
-Delete Volume
-
-```bash
-docker volume rm zwavejs2mqtt
-```
-
-# Custom builds
+# Custom Docker Builds
 
-There are two methods to build a custom docker container, using either the project's Dockerfile or Dockerfile.contrib. While the Dockerfile method offers a faster build process, the alternative Dockerfile.contrib method allows you to specify the specific branches for node-zwave-js and zwavejs2mqtt, and even the ability to specify alternative repositories (such as your own).
+There are two methods to build a custom docker container, using either the project's Dockerfile or Dockerfile.contrib. While the Dockerfile method offers a faster build process, the alternative Dockerfile.contrib method allows you to specify the specific branches for node-zwave-js and Zwavejs2Mqtt, and even the ability to specify alternative repositories (such as your own).
```

Comment:
You're not yet consistent between `Zwavejs2Mqtt` and `ZWavejs2Mqtt` (see link below)